src/Admin/Block/Extension/BlockCacheExtension.php

```diff
@@ -22,16 +22,37 @@
  */
 class BlockCacheExtension extends AbstractAdminExtension
 {
+    private $formGroup;
+    private $formTab;
+
+    public function __construct($formGroup = 'form.group_metadata', $formTab = 'form.tab_general')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     /**
      * Configure form fields.
      *
      * @param FormMapper $formMapper
      */
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
+
         $formMapper
-            ->with('form.group_metadata', array('translation_domain' => 'CmfBlockBundle'))
-                ->add('ttl', TextType::class)
+            ->tab($this->formTab, 'form.tab_general' === $this->formtab
+                ? ['translation_domain' => 'CmfSonataAdminIntegrationBundle']
+                : []
+            )
+                ->with($this->formGroup, 'form.group_metadata' === $this->formGroup
+                    ? ['translation_domain' => 'CmfSonataAdminIntegrationBundle']
+                    : []
+                )
+                    ->add('ttl', TextType::class)
```

Comment:
as you only set the translation domain conditionally on the tab + group, you should set the domain on every field in the admin extensions. can you check them again and add that please?